include/fft.hxx

```diff
@@ -78,4 +78,28 @@ void DST(const BoutReal *in, int length, dcomplex *out);
  */
 void DST_rev(dcomplex *in, int length, BoutReal *out);
 
+template <typename T>
+class Array;
+
+class Options;
+
+namespace bout {
+namespace fft {
+/// Should the FFT functions find and use an optimised plan?
+void fft_init(bool fft_measure);
+/// Should the FFT functions find and use an optimised plan?
+///
+/// If \p options is not nullptr, it should contain a bool called
+/// "fftw_measure". If it is nullptr, use the global `Options` root
+void fft_init(Options* options = nullptr);
+
+/// Returns the fft of a real signal \p in using fftw_forward
+Array<dcomplex> rfft(const Array<BoutReal>& in);
```

Comment:
Can we / should we move all fft routines into this namespace now and make the non-namespaced variants just call the namespaced version?